src/pages/index.js

```diff
@@ -1,13 +1,16 @@
 import React from "react";
 import Button from "../components/button/Button";
 import GameClock from "../components/gameclock/GameClock"
+import Team from "../components/team/Team"
```

Comment:
This should read `import Team from "../team" `because the team component would have had the index.js and the css.module.scss you do not need to name the javascript file Team